### ISSUE TITLE
Toasts not showing for me

### ISSUE THREAD

**Issue by acrigney** [gurrenm3/Toast-Notifications-with-.Net-Maui#1]
Thanks heaps for doing this great little project for us, except its not working for me, they did not work in my app when I did all the steps with VS2022 17.3.? (not sure what I had before) but now I have upgraded to VS2022 17.4.1 today and its not working again.
Also I had to add the Android SDK Platform 31 as after the upgrade the projects needed it.

**Comment by gurrenm3** [comment]
Hmmmm thats really weird. I'm no where near an expert on this so I don't have too much insight on it. My first suggestion would be to try making a brand new "throw away" Maui project and make sure to follow all of the steps exactly. I originally wrote the app for the .Net 6 version of Maui. Idk if there might be issues with the .Net 7 version if that's what you're on

**Comment by acrigney** [comment]
Thanks mate yes I did that a few times for both .net 6 and 7.

**Comment by gurrenm3** [comment]
Is it just not showing at all? Also try cloning this project and running it to see if it works at all. This project should work so if it doesn't it could be the PC

**Comment by acrigney** [comment]
Yes mate I did that and it won't work for .net 6 or .net 7 and I was working with Microsoft Chinese guys today and they could not get toasts to work with the Microsoft.Toolkit.UWP.Notifications assembly. They are going to chase it up in the US. Its probably an OS issue but the Microsoft guy had windows 11 enterprise.
